packages/sql/src/CopyFromManager.ts

```diff
@@ -18,7 +18,7 @@ export class CopyFromManager extends RequestManager {
       body: file,
     };
 
-    const customHeaders = event ? event.getHeaders() : [];
+    const customHeaders = options.event ? options.event.getHeaders() : [];
```

Comment:
I think we could use the null coalescing operator, At least I've used it before somewhere in this project.

Reply:
TS 3.7 for the win!